RecoTracker/LSTCore/standalone/SDL/Makefile

```diff
@@ -99,6 +100,15 @@ CUTVALUEFLAG_FLAGS = -DCUT_VALUE_DEBUG
 %_rocm.o: ../../src/alpaka/%.dev.cc
 	$(COMPILE_CMD_ROCM) $(CXXFLAGS_ROCM) $(ROOTINCLUDE) $(PRINTFLAG) $(CACHEFLAG) $(CUTVALUEFLAG) $(LSTWARNINGSFLAG) $(CMSSW_WERRORS_ROCM) $(T5CUTFLAGS) $(PTCUTFLAG) $(DUPLICATES) $(ALPAKAINCLUDE) $(ALPAKABACKEND_ROCM) $< -o $@
 
+%_cpu.o: ../../src/%.cc
```

Comment:
It's not ideal having to duplicate these targets. Does anyone know how to avoid this duplication?